src/config/index.ts

```diff
@@ -3,6 +3,11 @@ import { Configuration } from './config.types';
 
 const config = (): Configuration => ({
 	port: parseInt(process.env.PORT, 10) || DEFAULT_CONFIG.port,
+	host: process.env.HOST,
+	samlIdpMetaDataEndpoint: process.env.SAML_IDP_META_DATA_ENDPOINT,
```

Comment:
Deze env vars mag je nog mee in de `.env/.env.template` file steken, daar houden we een overzicht van alle variabelen zonder deze in te vullen.

Reply:
.env.template updated